scripts/vast-completions.bash

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+_vast_completions()
+{
+  completion_words=("${COMP_WORDS[@]}")
+  if [ "${COMP_WORDS[@]: -1}" == "--" ]; then
+    # We need to special-case '--', since that has special
+    # meaning, e.g.
+    #
+    #    vast pivot --asdf help -> displays error message/help text
+    #    vast pivot -- help     -> runs pivot command
+    #
+    # Technically we should scan the whole COMP_WORDS array
+    # for '--', but that issue should come up rarely, wheras
+    # a '--' at the end happens all the time as the result of
+    # a previous autocompletion.
```

Comment:
For zsh, this can be done using `if $(( COMP_WORDS[(Ie)--] )); then ... fi`, but I'm not quite sure that's as easy for bash. I still think this should be fixed, shouldn't be much more than a simple loop.